index.html

```diff
@@ -65,6 +65,215 @@ <h1>
+            <ul class="box-tags">
+                <li>Rubi on rails</li>
+                <li>css</li>
+                <li>JavaScript</li>
+                <li>html</li>
+            </ul>
+            <button class="button" type="button">
+                See Project
+            </button>
+        </article>
+    </section>
+    <section class="about-me">
+        <h1>About me</h1>
+        <p>
+            I can help you build a product , feature or website Look through some of my work and experience! If you like
+            what you see and have a project you need coded,
+            don’t hestiate to contact me.
+        </p>
+        <button class="button1" type="button">
+            Get My Resume
+        </button>
+        <ul class="languages-about-me">
+            <li>
+                <article class="box-languages">
+                    <div class="languages-image">
+                        <img class="img2" src="icon3.svg" alt="icon languages">
+                    </div>
+                    <h2 class="languages-title">
+                        Languages
+                    </h2>
+                    <ul class="languages-tags">
+                        
+                        <li>Rubi</li>
+                        <li>css</li>
+                        <li>JavaScript</li>
+                        <li>html</li>
+                    </ul>
+                </article>
+            </li>
+            <li>
+                <article class="box-languages">
+                    <div class="languages-image">
+                        <img class="img3" src="icon2.png" alt="icon languages">
+                    </div>
+                    <h2 class="languages-title">
+                        Frameworks
+                    </h2>
+                    <ul class="languages-tags">
+                         
+                        <li>Rubi</li>
+                        <li>Selenium</li>
+                        <li>Bootstrap</li>
+                        <li>Capybara</li>
+                        <li>RSpec</li>
+                    </ul>
+                </article>
+            </li>
+            <li>
+                <article class="box-languages">
+                    <div class="languages-image">
+                        <img class="img4" src="icon1.png" alt="icon languages">
+                    </div>
+                    <h2 class="languages-title">
+                        skills
+                    </h2>
+                    <ul class="languages-tags">
+                        <li>Codekit</li>
+                        <li> GitHub</li>
+                        <li>Codepen</li>
+                        <li>Gitlab</li>
+                        <li>Terminal</li>
+                    </ul>
+                </article>
+            </li>
+        </ul>
+    </section>
+    <footer class="footer-box">
+        <h1>
+            I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+        </h1>
```

Comment:
- [ ] The same thing for this title, It would be better to change the `h1` to `h2` or `h3` for this title.